backend/src/resolvers/posts.js

```diff
@@ -1,6 +1,43 @@
-import { neo4jgraphql } from 'neo4j-graphql-js'
+import { neo4jgraphql, cypherQuery } from 'neo4j-graphql-js'
+import { v1 as neo4j } from 'neo4j-driver'
+import _ from 'lodash'
+
+export function extractQueryResult ({ records }, returnType) {
+  const variableName = 'post'
+  let result = null
+  result = records.map(record => record.get(variableName))
+  // handle Integer fields
+  result = _.cloneDeepWith(result, field => {
+    if (neo4j.isInt(field)) {
+      // See: https://neo4j.com/docs/api/javascript-driver/current/class/src/v1/integer.js~Integer.html
+      return field.inSafeRange() ? field.toNumber() : field.toString()
+    }
+  })
+  return result
+}
 
 export default {
+  Query: {
+    Post: async (object, params, context, resolveInfo) => {
+      let [query, cypherParams] = cypherQuery(params, context, resolveInfo)
+
+      // HACK-ATTACK
+      query = query.replace('RETURN', 'WITH `post` ORDER BY `post`.createdAt DESC RETURN')
```

Comment:
Here's the hack :neckbeard:

Reply:
Let’s add a `// TODO: unshack hack-attack` 🦹🏼‍♂️🤣